src/random_test.hpp

```diff
@@ -42,6 +43,10 @@ std::string rand_float(float upper, float lower = 0);
 std::string rand_double(double upper, double lower = 0);
 std::string rand_string(int upper, int lower = 2);
 
+// Introduction of new function to retrive deque of recent queries
+std::deque<std::string> get_recent_queries();
```

Comment:
Isn't better to make it part of Thd class?

Reply:
Hmm, Removed and it is already part of Thd class